### PR TITLE
[FIX][12.0] base_changeset: fix write return type

### PR DESCRIPTION
**Pull request by dkubiak789**

### CODE REVIEW

**Comment by dkubiak789** [comment]
It returned a recordset instead of boolean.

File: base_changeset/models/base.py [hunk 1]
```diff
@@ -114,7 +114,7 @@ def write(self, values):
         for record in self:
             local_values = self.env["record.changeset"].add_changeset(record, values)
             super(Base, record).write(local_values)
-        return self
+        return bool(self)
```
Comment (StefanRijnhart) [inline base_changeset/models/base.py]: Good catch! I would simply return True as does the upstream write method: https://github.com/odoo/odoo/blob/12.0/odoo/models.py#L3403

Comment (StefanRijnhart) [reply base_changeset/models/base.py]: Also, please add the module name in the commit message and the PR title `[FIX] base_changeset: fix write return type`.

**Review by StefanRijnhart — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Merging, as it was included in later versions.

/ocabot merge patch

**Comment by StefanRijnhart** [comment]
Amazing, thanks @thomaspaulb

**Pull request merged**